src/index.js

```diff
@@ -1,36 +1,69 @@
 import './index.css';
+import ListControl from './modules/tasks';
+import ControlDisplay from './modules/display';
 
-const listData = [
-  {
-    description: 'Physical training',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Market',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Learning',
-    completed: false,
-    index: 2,
-  },
-];
-
-function tasktemplate(listItem) {
-  return `
-  <div class="line">
-    <input type="checkbox">
-    <p>${listItem.description}</p>
-    <i class="fas fa-ellipsis-v"></i>
-  </div>
-  `;
-}
-const rootList = document.querySelector('.list-container');
-
-listData.forEach((listItem) => {
-  const article = document.createElement('article');
-  article.innerHTML = tasktemplate(listItem);
-  rootList.appendChild(article);
+const selector = (element) => document.querySelector(element);
+const selectorAll = (element) => document.querySelectorAll(element);
+const input = selector('input');
+const manager = new ListControl();
+
+const updateTask = (event, index, focus = false) => {
+  if (event.key === 'Enter' || focus) {
+    manager.updateTask(
+      index,
+      selector(`.task_${index}`).value,
+      selector(`.task_${index}_checkbox`).checked,
+    );
+    selector('.line input').focus();
+  }
+};
+
+const removeList = (index) => {
+  manager.removeList(index);
+  ControlDisplay.reset(selector('.tasks'));
+  manager
+    .getTasks()
+    .forEach((task) => ControlDisplay.displayTask(selector('.tasks'), task));
```

Comment:
- [ ] **Hint:** Before page render you should check if the local storage contains any data that should be rendered, else render new data supplied by the user. I hope this makes sense 😄😄

Reply:
@brytebee This is also done thank you. Can you take a re-review?